paasta_tools/cli/cmds/status.py

```diff
@@ -259,7 +261,24 @@ def report_status_for_cluster(
                 for line in status.rstrip().split('\n'):
                     paasta_print('    %s' % line)
 
-    paasta_print(report_invalid_whitelist_values(instance_whitelist, seen_instances, 'instance'))
+            paasta_print('Remote runs:')
```

Comment:
This is a lot of shelling out and ssh'ing for something that won't have anything most of the time.

If someone runs 'paasta status -s yelp-main -i foo', why should they see a list of (unrelated) remote runs?